projects/plugins/jetpack/_inc/client/main.jsx

```diff
@@ -199,6 +200,87 @@ class Main extends React.Component {
 	}
 
 	renderMainContent = route => {
+		/**
+		 * There are two conditions (groups of conditions, really) where we would want to
+		 * show the partner connection screen.
+		 *
+		 * 1. The site is not yet connected to WPCOM, but has the jetpack_partner_coupon
+		 * option set in the database (this.props.partnerCoupon in redux). This is likely a
+		 * partner-user who has just arrived here from a CTA within a partner's dashboard
+		 * or other ecosystem.
+		 *
+		 * 2. The site is already connected to WPCOM, but the jetpack_partner_coupon option
+		 * is still set in the database. This means the user connected their site, but never
+		 * redeemed the coupon. If this is the case, we don't want to override the dashboard
+		 * or at a glance pages with the redemption screen. Instead, we'll catch a URL
+		 * parameter that JITMs will set (showCouponRedemption=true), and show the screen only
+		 * when the user came from a a JITM.
+		 */
+		if ( this.props.partnerCoupon ) {
+			const forceShow = new URLSearchParams( window.location.search ).get( 'showCouponRedemption' );
+			const partnerCoupon = this.props.partnerCoupon;
+
+			if ( ! this.props.isSiteConnected || forceShow ) {
+				this.partnerCouponHandleClick = () => {
+					window.location.href = getRedirectUrl( 'jetpack-plugin-partner-coupon-checkout', {
+						path: partnerCoupon.product.slug,
+						site: this.props.siteRawUrl,
+						query: `coupon=${ partnerCoupon.coupon_code }`,
+					} );
+				};
+
+				return (
+					<ConnectScreen
+						apiNonce={ this.props.apiNonce }
+						registrationNonce={ this.props.registrationNonce }
+						apiRoot={ this.props.apiRoot }
+						images={ [ '/images/products/illustration-backup.png' ] }
```

Comment:
The plugin might be installed in a sub-directory which would break the relative path (`https://example.com/installation/`).

Due to the relative URL, browsers would try and look for the image at:
`https://example.com/images/products/illustration-backup.png`
and not:
`https://example.com/installation/images/products/illustration-backup.png`

Using `import { imagePath } from 'constants/urls'` should be a pretty quick fix:
```
images={ [ `${ imagePath }products/illustration-backup.png` ] }
```

Reply:
My bad, it seems like this is handled in `ImageSlider` where it combines them `assetBaseUrl + image`. Not very well documented, but hey! It works!